<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportOptionsWidget</class>
 <widget class="QWidget" name="ImportOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import to the Database Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblBaseFolder">
       <property name="text">
        <string>Destination folder:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leBaseFolder"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="gbFilesAndFolders">
     <property name="title">
      <string>Files and folders options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="cbRecursively">
        <property name="text">
         <string>Process directories recursively</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbKeepStructure">
        <property name="text">
         <string>Keep folders structure</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbCreateSubfoldersForTopLevelFolder">
        <property name="text">
         <string>Create a subfolder for the top level folder</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbCreateSubfoldersForFiles">
        <property name="text">
         <string>Create a subfolder for each file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbImportUnrecognized">
        <property name="text">
         <string>Import unrecognized files</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="gbMultiSequencePolicy">
        <property name="title">
         <string>Multi-sequence files import policy</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QRadioButton" name="rbSeparate">
           <property name="text">
            <string>Import as separate sequences</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbMerge">
           <property name="text">
            <string>Merge into a single sequence</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="layoutMerge">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="lblMerge">
             <property name="text">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Number of 'unknown' symbols &lt;/span&gt;&lt;span style=&quot; font-size:10pt; font-style:italic;&quot;&gt;('N 'for nucleic or 'X' for amino)&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt; between parts:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sbMerge">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="suffix">
              <string> bases</string>
             </property>
             <property name="maximum">
              <number>1000000</number>
             </property>
             <property name="value">
              <number>10</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QRadioButton" name="rbMalignment">
           <property name="text">
            <string>Join into alignment</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="gbAcePolicy">
        <property name="title">
         <string>Import ACE files as</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QRadioButton" name="rbAceAsMalignment">
           <property name="text">
            <string>Multiple sequence alignment</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbAceAsAssembly">
           <property name="text">
            <string>Short reads assembly</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbDocsAndObjects">
     <property name="title">
      <string>Documents and objects options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="cbCreateSubfoldersforDocs">
        <property name="text">
         <string>Create a subfolder for each document</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>rbMerge</sender>
   <signal>toggled(bool)</signal>
   <receiver>sbMerge</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
